code/modules/mob/living/carbon/human/human.dm

```diff
@@ -66,6 +66,7 @@
 
 /mob/living/carbon/human/Destroy()
 	QDEL_LIST(bodyparts)
+	QDEL_LIST(splinted_limbs)

```

Comment:
Do I need to do splinted_list.Cut() then? Wouldn't the references already be gone since the limbs are deleted?